examples/addons/aad-pod-identity/README.md

```diff
@@ -0,0 +1,67 @@
+# AAD Pod Identity Add-on
+
+
+This is the AAD Pod Identity add-on.  Add this add-on to your json file as shown below to automatically enable AAD Pod identity in your new Kubernetes cluster.
+
+```json
+{
+    "apiVersion": "vlabs",
+    "properties": {
+      "orchestratorProfile": {
+        "orchestratorType": "Kubernetes",
+        "kubernetesConfig": {
+        "addons": [
+          {
+            "name": "aad-pod-identity",
+            "enabled": true
+          }
+        ]
+      }
+      },
+      "masterProfile": {
+        "count": 1,
+        "dnsPrefix": "",
+        "vmSize": "Standard_DS2_v2",
+      },
+      "agentPoolProfiles": [
+        {
+          "name": "agentpool",
+          "count": 3,
+          "vmSize": "Standard_DS2_v2",
+          "availabilityProfile": "AvailabilitySet"
+        }
+      ],
+      "linuxProfile": {
+        "adminUsername": "azureuser",
+        "ssh": {
+          "publicKeys": [
+            {
+              "keyData": ""
+            }
+          ]
+        }
+      },
+      "servicePrincipalProfile": {
+        "clientId": "",
+        "secret": ""
+      }
+    }
+  }
+
+```
+
+You can validate that the add-on is running as expected with the following commands:
+
+You should see two pod names - one starting with mic and another nmi as `Running` after executing:
```

Comment:
`You should see two sets of pods - starting with a single mic pod and as many nmi pod as there are agent nodes in `Running` state after executing:`